lib/fathead/react_native/output.txt

```diff
@@ -0,0 +1,1410 @@
+ActionSheetIOS	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Display an iOS action sheet. The options object must contain one or more of:</p></section>	https://facebook.github.io/react-native/releases/0.40/docs/actionsheetios.html#
+ActionSheetIOS ActionSheetIOS.showActionSheetWithOptions	R	ActionSheetIOS.ActionSheetIOS.showActionSheetWithOptions										
```

Comment:
Actually, is this duplication intended to be there? It looks very redundant, and I doubt the user would type out the same name twice + method.

The docs only show the method name, so it'l likely a user would just search that as well.

I wonder if it would be better for the articles to just be the method names